extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -406,6 +413,7 @@ class QueueProcessor extends EventEmitter {
             groupId,
             concurrency: this.repConfig.queueProcessor.concurrency,
             queueProcessor: queueProcessorFunc,
+            logConsumerMetricsIntervalS: this.repConfig.queueProcessor.logConsumerMetricsIntervalS,
             canary: true,
             circuitBreaker: this.circuitBreakerConfig,
```

Comment:
I just noticed that we also have this in bucket notification's queue processor. I think we should add the same logic there too.